src/main/java/io/dropwizard/revolver/resource/RevolverCallbackResource.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2016 Phaneesh Nagaraja <phaneesh.n@gmail.com>.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package io.dropwizard.revolver.resource;
+
+import com.codahale.metrics.annotation.Metered;
+import com.google.common.base.Strings;
+import io.dropwizard.msgpack.MsgPackMediaType;
+import io.dropwizard.revolver.base.core.RevolverCallbackResponse;
+import io.dropwizard.revolver.callback.InlineCallbackHandler;
+import io.dropwizard.revolver.http.RevolverHttpCommand;
+import io.dropwizard.revolver.persistence.PersistenceProvider;
+import io.dropwizard.revolver.util.HeaderUtil;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import javax.inject.Singleton;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+
+/**
+ * @author phaneesh
+ */
+@Path("/revolver")
```

Comment:
@himadrisj We won't need this as this would get imported from revolver bundle once we update the version which is supported in api